src/test/java/org/folio/rest/impl/LostItemFeePoliciesAPITest.java

```diff
@@ -146,7 +148,10 @@ private static JsonObject createEntityJson() {
       .put("description", "This is description for undergrad standard")
       .put("itemAgedLostOverdue", new JsonObject().put("duration", 12).put("intervalId", "Months"))
       .put("patronBilledAfterAgedLost",
-        new JsonObject().put("duration", 12).put("intervalId", "Months"))
+        new JsonObject().put("duration", 5).put("intervalId", "Months"))
+      .put("recalledItemAgedLostOverdue", new JsonObject().put("duration", 12).put("intervalId", "Months"))
+      .put("patronBilledAfterRecalledItemAgedLost",
+        new JsonObject().put("duration", 6).put("intervalId", "Months"))
```

Comment:
Do we need assertion statements to verify the properties got persisted?

Reply:
That was my initial thought, but looking at the test, it doesn't seem to explicitly check for persistence of *any* specific attributes.  My guess based on the tinkering I did is that the original test writer assumed that if it couldn't accept an attribute, it would kick back a 422 error, so they just checked for successful creation of the policy as a whole.  If we are going to add a check for these specific attributes, we should also add a check for all of the others.

Reply:
Sounds good. Yeah, the API should return 422 in case of unknown property.